docs/build/references/command-line-interface.md

```diff
@@ -20,6 +20,14 @@ Example JSON config file:
 }
 ```
 
+`--config-file-content` (string):
+
+As an alternative to `--config-file`, it allows specifying base64 encoded config content. Must be used in conjunction with `--config-file-content-type`.
+
+`--config-file-content-type` (string):
+
+Specifies the format of the base64 encoded config content. Required if `--config-file-content` is set.
```

Comment:
Might be useful to list which types are allowed. Might just be my ignorance, but without looking into the source it's not entirely clear what should those files types be. Is this like JSON/YAML and such?

Reply:
We currently use [viper ](https://github.com/spf13/viper#reading-config-files) to parse configs, so we can handle JSON, TOML, YAML among others.  I included the viper link to show full list of supported types.